docs/governance/templates/release_issue.md

```diff
@@ -27,75 +27,70 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Review closed issues in the current milestone to ensure that they have appropriate tags.
 - [ ] Review [merged PRs that have no milestone](https://github.com/googleforgames/agones/pulls?q=is%3Apr+is%3Amerged+no%3Amilestone+) and add them to the current milestone.
 - [ ] Review merged PRs in the current milestone to ensure that they have appropriate tags.
-- [ ] Ensure the next RC and stable releases in the Google Calendar have the correct version number.
+- [ ] Ensure the next stable releases in the Google Calendar have the correct version number.
 - [ ] Ensure the next version milestone is created.
 - [ ] Any issues in the current milestone that are not closed, move to next milestone.
-- [ ] If release candidate add the label `feature-freeze-do-not-merge` to any feature pull requests.
 - [ ] `git checkout main && git pull --rebase upstream main`
-- [ ] If full release, run `make release-deploy-site`
-- [ ] Run `make build-release` to generate the CHANGELOG.md (if release candidate
-  `RC_RELEASE=1 make build-release`). You will need your
+- [ ] Run `make release-deploy-site`
+- [ ] Run `make build-release` to generate the CHANGELOG.md. You will need your
   [GitHub Personal Access Token](https://github.com/settings/tokens) for this.
```

Comment:
```suggestion
- [ ] Run `make build-release` to generate the CHANGELOG.md.
```

Since this runs in Cloud Build, we don't need the GitHub token anymore so we can clean this up too.

Reply:
Thanks for the review, Mark. I have removed the token info.